### ISSUE TITLE
Psychose

### ISSUE THREAD

**Issue by daclavel** [yaucheres/puckmanager#3]
Heu, juste une petite précision...
Est ce que tu as paramétrer ce github en mode privé?
D

**Comment by yaucheres** [comment]
non, mais on s'en fout, il n'y a pas de données particulière dedans ?

**Comment by daclavel** [comment]
Est ce aue l'on peut quand meme le mettre en prive??

**Comment by yaucheres** [comment]
et surtout parce que c'est payant :-)

**Comment by daclavel** [comment]
ah l'argument de choix!

**Comment by daclavel** [comment]
je modifierais le fichier index.htmal alors!!
:)
